web3deployment/reserveDeploy/fpr/enhancedStepsDeployer.js

```diff
@@ -0,0 +1,76 @@
+const KyberReserveHighRate = artifacts.require("KyberReserveHighRate.sol");
+const ConversionRateEnhancedSteps = artifacts.require("ConversionRateEnhancedSteps.sol");
+const WrapConversionRateEnhancedSteps = artifacts.require("WrapConversionRateEnhancedSteps.sol");
+
+const BN = web3.utils.BN;
+
+let reserve;
+let reserveAddr = "";
+let conversionRate;
+let conversionRateAddr = "";
+let wrapper;
+let wrapperAddr = "";
+
+let networkAddr = "0x920b322d4b8bab34fb6233646f5c87f87e79952b";
+let admin= "0xf3D872b9E8d314820dc8E99DAfBe1A3FeEDc27D5";
+let deployer;
+
+async function main() {
+    const accounts = await web3.eth.getAccounts();
+    deployer = accounts[0];
```

Comment:
plan is to send funds to this address?
if so how to return back remaining funds when done?
have private key?
consider saving it locally.

Reply:
if you set the network is ropsten in the script, accounts[0] will be the address from the privatekey in .env file